scene/gui/rich_text_label.cpp

```diff
@@ -2788,7 +2788,7 @@ void RichTextLabel::gui_input(const Ref<InputEvent> &p_event) {
 			menu->set_position(get_screen_transform().xform(b->get_position()));
 			menu->reset_size();
 			menu->popup();
-			grab_focus();
+			grab_focus(true);
```

Comment:
Why it's trying to grab focus at all, I suspect it's a typo and should actually give focus to the menu.

```suggestion
			menu->grab_focus();
```

Reply:
After examining the context menu invocations in other locations, I think you're right; it should be `menu->grab_focus()` just like elsewhere.